single/docker-compose.yml

```diff
@@ -4,9 +4,10 @@ services:
     image: nutsfoundation/nuts-node:master
     environment:
       NUTS_CONFIGFILE: /opt/nuts/nuts.yaml
+      NUTS_NETWORK_BOOTSTRAPNODES: ""
```

Comment:
in the workshop, we'll change the config file. This will override that?

Reply:
Ah I assumed that in the workshop the docker-compose file would be edited, so I added this as convenience for the participants, so they can't make mistakes on how to spell/underscore it. But in that case I'll remove it.